rules/integrations/aws/lateral_movement_aws_ssm_start_session_to_ec2_instance.toml

```diff
@@ -0,0 +1,111 @@
+- **Review Session Details**: Examine details like the session ID and stream URL (`aws.cloudtrail.flattened.response_elements`) to understand the scope and nature of the session.
+    - Check if any commands executed during the session were unauthorized or out of ordinary practices.
+- **Correlate with Other Security Events**: Look for other related security events around the time of the session start to identify any pattern or broader attack vector that may involve this user or EC2 instance.
+
+### False Positive Analysis
+
+- **Legitimate Administrative Activities**: Confirm whether the SSM session was initiated for valid administrative purposes such as system maintenance, patching, or configuration updates. Verify with the respective teams or personnel.
+
+### Response and Remediation
+
+- **Immediate Session Review**: If the session initiation seems suspicious, review all actions taken during the session.
+    - If possible, terminate the session to prevent any potential harm.
+- **Validate and Reinforce Security Policies**: Ensure that policies around SSM session initiation are strict and adhere to the principle of least privilege.
+    - Update IAM policies if necessary to tighten controls.
+- **Incident Response Activation**: If malicious intent or actions are confirmed, activate the incident response protocol.
+    - This includes containment of the threat, eradication of the adversary’s presence, recovery of affected systems, and a thorough investigation.
+- **Enhance Monitoring and Alerts**: Improve monitoring of SSM sessions, particularly focusing on sessions that involve sensitive or critical EC2 instances.
+    - Adjust alerting mechanisms to flag unusual session initiations promptly.
+
+### Additional Information
+
+For more in-depth understanding of managing SSM sessions and security best practices, refer to the [AWS Systems Manager documentation](https://docs.aws.amazon.com/systems-manager/latest/APIReference/API_StartSession.html). Additionally, consider the security implications and best practices outlined in [AWS SSM privilege escalation techniques](https://cloud.hacktricks.xyz/pentesting-cloud/aws-security/aws-privilege-escalation/aws-ssm-privesc).
+
+"""
+references = [
+    "https://docs.aws.amazon.com/systems-manager/latest/APIReference/API_StartSession.html",
+    "https://hackingthe.cloud/aws/post_exploitation/intercept_ssm_communications/",
+]
+risk_score = 47
+rule_id = "804a7ac8-fc00-11ee-924b-f661ea17fbce"
+severity = "medium"
+tags = [
+    "Domain: Cloud",
+    "Data Source: AWS",
+    "Data Source: Amazon Web Services",
+    "Data Source: AWS SSM",
+    "Use Case: Threat Detection",
+    "Tactic: Lateral Movement",
+]
+timestamp_override = "event.ingested"
+type = "new_terms"
+
+query = '''
+event.dataset: "aws.cloudtrail" and event.provider: "ssm.amazonaws.com"
+    and event.action: "StartSession" and event.outcome:success
```

Comment:
I am being picky here, but we should attempt consistency across KQL queries. Either use quotes or not, and either use spaces or not. I see different syntax across all of the rules. It's fine now, but we should probably discuss best practices with the team.

Reply:
fixed!